### PR TITLE
Show namespace creation date

### LINKED ISSUE #202: Display the creation time for namespaces
That is, on the namespaces page

### PR DESCRIPTION
**Pull request by lsamayoa**
Fixes #202

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,4 @@
 /coverage
 /public/assets
 /config/config-local.yml
+.DS_Store
```
Comment (mssola) [inline .gitignore]: This is MacOS X specific. Please, put it in your global gitignore :)

**Comment by mssola** [comment]
Just fix that small thing. Thanks! :+1:

**Comment by lsamayoa** [comment]
Done

**Comment by mssola** [comment]
Thanks @lsamayoa ! :)

**Pull request merged**